sql/core/src/test/scala/org/apache/spark/sql/CachedTableSuite.scala

```diff
@@ -33,7 +33,9 @@ import org.apache.spark.sql.execution.exchange.ShuffleExchangeExec
 import org.apache.spark.sql.functions._
 import org.apache.spark.sql.internal.SQLConf
 import org.apache.spark.sql.test.{SharedSQLContext, SQLTestUtils}
+import org.apache.spark.sql.types.{StringType, StructField, StructType}
 import org.apache.spark.storage.{RDDBlockId, StorageLevel}
+import org.apache.spark.storage.StorageLevel.{apply => _, _}
```

Comment:
Sorry for nit-picking. But, we only use two memory levels here.
```scala
-import org.apache.spark.storage.StorageLevel.{apply => _, _}
+import org.apache.spark.storage.StorageLevel.{MEMORY_AND_DISK_2, MEMORY_ONLY}
```

Reply:
Yes. It touches only two levels. I also updated this import.